books/models.py

```diff
@@ -0,0 +1,59 @@
+from django.db import models
+
+from utils        import timer
+from users.models import User
+
+class Book(timer):
+    name          = models.CharField(max_length=200)
+    price         = models.DecimalField(max_digits=10, decimal_places=2)
+    preview       = models.URLField(max_length=1000)
+    fullview      = models.URLField(max_length=1000)
```

Comment:
view가 아니라 file 아닌가요?

```suggestion
    preview_file       = models.URLField(max_length=1000)
    file = models.URLField(max_length=1000)
```